README.md

```diff
@@ -121,4 +122,4 @@ This project is licensed under the Apache-2.0 License.
 ## Issues and Contributing
```

Comment:
QuickStart have been moved to docs/QuickStart.md in ad46f147205ce0ef722cb66feff1ff2f21af5680